docs: update core documentation for Turbopack as default bundler (#84282)
vercel/next.js: docs/01-app/01-getting-started/01-installation.mdx

These scripts refer to the different stages of developing an application:

- `next dev`: Starts the development server using Turbopack (default bundler).
- `next build`: Builds the application for production.
- `next start`: Starts the production server.
- `eslint`: Runs ESLint.

> Turbopack is now the default bundler, to use Webpack instead of Turbopack for development, use `next dev --webpack`. See the [Turbopack docs](/docs/app/api-reference/turbopack) information on configuration.

Reviewer: Does this apply to build too? As in `Turbopack` is the default in both, and to use Webpack `--webpack` can be passed in.  

```suggestion
Turbopack is now the default bundler. To use Webpack run `next dev --webpack` or `next build --webpack`. See the [Turbopack docs](/docs/app/api-reference/turbopack) for configuration details.
```

Author: good catch. yes default for dev and build
